worker/src/main/java/dk/dbc/search/work/presentation/worker/Config.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (C) 2020 DBC A/S (http://dbc.dk/)
+ *
+ * This is part of work-presentation-worker
+ *
+ * work-presentation-worker is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * work-presentation-worker is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package dk.dbc.search.work.presentation.worker;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.util.Arrays;
+import java.util.Map;
+import javax.annotation.PostConstruct;
+import javax.ejb.EJBException;
+import javax.ejb.Lock;
+import javax.ejb.LockType;
+import javax.ejb.Startup;
+import javax.enterprise.context.ApplicationScoped;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.ClientRequestContext;
+import javax.ws.rs.client.ClientRequestFilter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ *
+ * @author Morten Bøgeskov (mb@dbc.dk)
+ */
+@ApplicationScoped
+@Startup
+@Lock(LockType.READ)
+public class Config {
```

Comment:
Kunne der være en forklaring af hvorfor vi ikke kan bruge @ConfigProperties?

Reply:
Jeg ved godt dét er politikken, det gør bare at man laver integrations test hvor properies bliver sat forskelligt i de forskellige klasser (forskellig naming-convension, url med eller uden traillig slash), og den resulterende applikation virker kun hvis ting er konfigureret på en måde det ene sted og en anden et andet sted - Den slags har jeg brugt alt for lang tid på.
Desuden skal der efterfølgende valideres at konfigurationerne er gyldige, det betyder at du kan få 2 forskellige kilder til konfigurationfejl.

Reply:
Interessant - burde det feedback ikke næsten gå til Jan, så, så det evt. kunne blive en del af templaten/politikken? Jo ærgerligt, hvis vi har politikker der er kontraproduktive.